.ci/win_devops_tests.bat

```diff
@@ -0,0 +1,3 @@
+python .ci\devops_setup_worker.py > worker_path
+set /p PYAZURE_WORKER_DIR=<worker_path
+python setup.py test
```

Comment:
Missed defining environment variables? Similar to linux_devops_tests.sh?

Reply:
This is fine. We don't need to export environment variables for Windows. Linux requires exporting environment variables so child processes can use them.